data/org.freedesktop.impl.portal.Screenshot.xml

```diff
@@ -52,6 +54,15 @@
               Defaults to no.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>permission_store_checked b</term>
+            <listitem><para>
+              Hint whether the screenshot portal has the 'screenshot' permission. Defaults
+              to no.
```

Comment:
Sorry, missed this but the portal doesn't have the permission, but it has checked the permission. I suggest
```
              Hint whether the screenshot portal has checked the 'screenshot' permission for
              the requesting app. Defaults to no.
```